DcShare/DcShare.entitlements

```diff
@@ -6,5 +6,9 @@
 	<array>
 		<string>group.chat.delta.ios</string>
 	</array>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)group.chat.delta.ios</string>
```

Comment:
is `$(AppIdentifierPrefix)` equal to `8Y86453UA8` from above?

Reply:
yes